### ISSUE TITLE
Dark theme is unusable

### ISSUE THREAD

**Issue by saibotsivad** [storybooks/storybook#5149]
**Describe the bug**

When I visit "Live Examples > 4.0 > React Official" the font/input element contrast is atrocious, rendering the whole example website completely unusable.

I poked around a bit but couldn't find anything in the code, did someone forget to include the lightening parts of the dark theme?

**To Reproduce**

1. Go to https://storybook.js.org/basics/live-examples/
2. Click on any of the 4.0 examples that use the dark theme (React, specifically)
3. Wait for page to load
4. Try looking at the "Components > Form > Select" as a specific example

**Expected behavior**

I should be able to read the text of the select element.

**Screenshots**

Firefox 64.0

![screenshot 2](https://user-images.githubusercontent.com/91254/50711847-14377280-1035-11e9-9ba2-6508afa94095.png)

Chrome 71.0.3578.98

![screenshot 4](https://user-images.githubusercontent.com/91254/50711952-6aa4b100-1035-11e9-909b-b47fda0636a3.png)

**System:**
 - OS: Windows 10
 - Device: HP Pavilion
 - Browser: Chrome and Firefox (versions noted above)
 - Version: 4.0

**Comment by shilman** [comment]
Fixed in v5
